### ISSUE TITLE
Can not parse JSON value with comma charactor

### ISSUE THREAD

**Issue by YeSei** [grafana/grafana-json-datasource#158]
## Problem
 if the api return a JSON object which have a value with comma charactor. the grafana-json-api plugin can not parse the value。
## The JSON object in the swagger:
```
{
  "code": 200,
  "message": "SUCCESS",
  "data": {
    "tagStr1": "entity,is_new",
    "tagStr3": "test",
    "tagtest2": 12
  }
}
```
## The JSON object in the grafana-json-api plugin:
![image](https://user-images.githubusercontent.com/39610341/129467250-8d20c9f6-a085-4527-97c1-4dd0131721bd.png)

The value in the JSON object which fieldPath is "data.tagStr1" is null.
Am I config something wrong?

## workspace:
JSON API version: 1.2.1
Grafana version: 7.5.5

**Comment by ZuperZee** [comment]
Try changing `$.data.tagStr1` to `$.tagStr1`

**Comment by marcusolsson** [comment]
Tested with both 7.5.5 and 8.1.2 and this seems to be working for me. Your configuration looks good to me though. You could try with the recently released 1.3.0 which had some bug fixes, but not sure if they're related.

<img width="844" alt="Screenshot 2021-09-03 at 19 45 04@2x" src="https://user-images.githubusercontent.com/8396880/132046523-e2a1c4df-1043-481a-9375-b9987eeb2616.png">

**Comment by gabor** [comment]
as we got no foeedback, we're closing the issue. we can reopen it if we receive new info.
